src/rpc/rawtransaction.cpp

```diff
@@ -644,7 +648,10 @@ static UniValue decoderawtransaction(const JSONRPCRequest& request)
     }
 
     UniValue result(UniValue::VOBJ);
-    TxToUniv(CTransaction(std::move(mtx)), uint256(), result, false);
+    auto tx = CTransaction(std::move(mtx));
+    vaulttxnstatus txStatus = TX_UNKNOWN;
+    vaulttxntype txType = GetVaultTxTypeNonContextual(tx);
+    TxToJSON(tx, uint256(), txType, txStatus, result);
```

Comment:
do `TxToJSON` and `TxToUniv` both return JSON?

Reply:
Yup, TxToJSON even call TxToUniv internally at the beginning.